Tasks/PublishSymbols/task.json

```diff
@@ -2,34 +2,36 @@
     "id": "0675668A-7BBA-4CCB-901D-5AD6554CA653",
     "name": "PublishSymbols",
     "friendlyName": "Index Sources & Publish Symbols",
-    "description": "Index your source code and publish symbols to a file share",
-    "helpMarkDown": "Index your source code and publish symbols to a file share<br/>[More Information](https://go.microsoft.com/fwlink/?LinkID=613722)",
+    "description": "Index your source code and publish symbols to a file share or Visual Studio Team Services Symbol Server",
+    "helpMarkDown": "See [more information](https://go.microsoft.com/fwlink/?LinkID=613722) on how to use this task. The Visual Studio Team Services Symbol Server is in private preview. See [these instructions](https://go.microsoft.com/fwlink/?linkid=846265) on how to participate in the private preview.",
     "category": "Build",
     "visibility": [
         "Build"
     ],
     "author": "Microsoft Corporation",
+    "preview": true,
     "version": {
-        "Major": 1,
-        "Minor": 0,
-        "Patch": 36
+        "Major": "2",
```

Comment:
Since you are bumping the major version, now is the time to move off of the legacy find implementation. In the box tasks are converging on [this implementation](https://github.com/Microsoft/vsts-task-lib/blob/master/powershell/Docs/FindingFiles.md). We need to move to the same globbing rules for in the box tasks, but since it breaks compat and you are bumping the major version now is the right time for this task. It should largely be a matter of calling one cmdlet instead of another. One of the functions is missing for the ps3 lib. If it's the one you need (whichever one fits the pattern for your task) then I'll add it.